common/common.cpp

```diff
@@ -2039,8 +2039,16 @@ std::string fs_get_cache_file(const std::string & filename) {
 //
 // Model utils
 //
+struct llama_init_result llama_init_default_params() {
```

Comment:
I don't think this is needed. We have `*_default_params` functions in case we want to allow user to create their own struct with default params, which is not the case here (`llama_init_result` is created by us, **not** by user)

Reply:
Thanks, i will correct it.